pkg/resources/server.go

```diff
@@ -289,6 +290,15 @@ func (rs *resourceServer) ListAndWatch(e *pluginapi.Empty, s pluginapi.DevicePlu
 			// FIXME: there is no way to recover from the Unhealthy state.
 			d.Health = pluginapi.Unhealthy
 			_ = s.Send(&pluginapi.ListAndWatchResponse{Devices: rs.devs})
+		case <-rs.updateSignal:
+			log.Printf("Updating \"%s\" devices", rs.resourceName)
+			resp.Devices = rs.devs
+
+			if err := s.Send(resp); err != nil {
```

Comment:
i dont think we want to fail here, just log and retry

Reply:
changed